misc/test/examples_test.go

```diff
@@ -103,20 +105,52 @@ func TestExamples(t *testing.T) {
 				})
 			},
 		}),
+	}
+
+	// The tests in the skippedExamples collection below will not be run and will be reported as SKIPPED on the test run
+	// output.  Put a comment above the test to document why the test is being skipped.
+	skippedExamples := []integration.ProgramTestOptions{
+		// This test fails due to a bug in the Terraform Azure provider in which the
+		// service principal is not available when attempting to create the K8s cluster.
+		// See the azure-ts-aks-example readme for more detail.
+		base.With(integration.ProgramTestOptions{
+			Dir:       path.Join(cwd, "..", "..", "azure-ts-aks-mean"),
+			SkipBuild: true,
+			Config: map[string]string{
+				"azure:environment": azureEnviron,
+				"password":          "testTEST1234+_^$",
+				"sshPublicKey":      "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDeREOgHTUgPT00PTr7iQF9JwZQ4QF1VeaLk2nHKRvWYOCiky6hDtzhmLM0k0Ib9Y7cwFbhObR+8yZpCgfSX3Hc3w2I1n6lXFpMfzr+wdbpx97N4fc1EHGUr9qT3UM1COqN6e/BEosQcMVaXSCpjqL1jeNaRDAnAS2Y3q1MFeXAvj9rwq8EHTqqAc1hW9Lq4SjSiA98STil5dGw6DWRhNtf6zs4UBy8UipKsmuXtclR0gKnoEP83ahMJOpCIjuknPZhb+HsiNjFWf+Os9U6kaS5vGrbXC8nggrVE57ow88pLCBL+3mBk1vBg6bJuLBCp2WTqRzDMhSDQ3AcWqkucGqf dremy@remthinkpad",
+			},
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				assertHTTPResult(t, stack.Outputs["endpoint"], func(body string) bool {
+					return assert.Contains(t, body, "<title>Node/Angular Todo App</title>>")
+				})
+			},
+		}),
 		// TODO[pulumi/pulumi#1606] This test is failing in CI, disabling until this issue is resolved.
-		// base.With(integration.ProgramTestOptions{
-		// 	Dir:           path.Join(cwd, "..", "..", "aws-py-webserver"),
-		// 	Verbose:       true,
-		// 	DebugLogLevel: 8,
-		// 	DebugUpdates:  true,
-		// 	SkipBuild:     true,
-		// 	Config: map[string]string{
-		// 		"aws:region": awsRegion,
-		// 	},
-		// 	ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
-		// 		expectHelloWorld(t, stack.Outputs["public_dns"])
-		// 	},
-		// }),
+		base.With(integration.ProgramTestOptions{
+			Dir:           path.Join(cwd, "..", "..", "aws-py-webserver"),
+			Verbose:       true,
+			DebugLogLevel: 8,
+			DebugUpdates:  true,
+			SkipBuild:     true,
+			Config: map[string]string{
+				"aws:region": awsRegion,
+			},
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				assertHTTPHelloWorld(t, stack.Outputs["public_dns"])
+			},
+		}),
+	}
+
+	// Print skipped example tests to the test output
+	if len(skippedExamples) > 0 {
```

Comment:
I don't think we need this - just commented with a TODO is fine.

Reply:
got it.  backed it out, moved to comment and TODO